src/main/java/com/onarandombox/multiverseinventories/share/Sharables.java

```diff
@@ -731,7 +730,8 @@ public static void setPlayerSpawnLocation(Player player, Location spawnLocation)
      */
     public static final SharableGroup ALL_DEFAULT = new SharableGroup("all", fromSharables(HEALTH, ECONOMY,
             FOOD_LEVEL, SATURATION, EXHAUSTION, EXPERIENCE, TOTAL_EXPERIENCE, LEVEL, INVENTORY, ARMOR, BED_SPAWN,
-            MAXIMUM_AIR, REMAINING_AIR, FALL_DISTANCE, FIRE_TICKS, POTIONS, LAST_LOCATION, ENDER_CHEST, OFF_HAND),
+            MAXIMUM_AIR, REMAINING_AIR, FALL_DISTANCE, FIRE_TICKS, POTIONS, LAST_LOCATION, ENDER_CHEST, OFF_HAND,
+            SPAWN_LOCATION),
             "*", "everything");
 
 
```

Comment:
This is the relevent code.

Reply:
Actually, it's not showing correctly, the relevent code is from line 757 to line 768.